ui/src/app/settings/components/Routes/Routes.tsx

```diff
@@ -33,18 +33,26 @@ import UsersList from "app/settings/views/Users/UsersList";
 const Routes = (): JSX.Element => {
   return (
     <Switch>
-      <Route exact path={settingsURLs.configuration.general} render={General} />
```

Comment:
These routes also needed to be changed to render the component without passing props through the route.